api/tests/routes/backoffice_v3/offerers_test.py

```diff
@@ -395,6 +414,7 @@ def test_get_stats(self, authenticated_client, offerer, offer, booking):
         # cast to integer to avoid errors due to amount formatting
         assert str(int(booking.amount)) in content
         assert "1 IND" in content  # one active individual offer
+        assert "2 EAC" in content  # two active collective offers (with collective offer template)
```

Comment:
C'est un peu basique de vérifier uniquement ainsi, qui ne vérifie pas que c'est dans le bon texte (offres actives).
Je propose de remplacer ces vérificatons par :
```suggestion
        cards_text = html_parser.extract_cards_text(response.data)
        assert "3 offres actives (1 IND / 2 EAC) 0 offre inactive (0 IND / 0 EAC)" in cards_text
```
Le CA au-dessus pourrait être testé de la même manière, puisque fait partie des `cards_text`.

Reply:
Done :heavy_check_mark: